tests/subsys/net/lib/nrf_cloud/prj.conf

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+
+# ZTest
+CONFIG_ZTEST=y
+CONFIG_ZTEST_NEW_API=y
+
+# Dependencies
+CONFIG_NEWLIB_LIBC=y
+CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y
+CONFIG_FPU=y
+CONFIG_CJSON_LIB=y
+
+# Mocking
+CONFIG_UNITY=y
+
+# MQTT
+CONFIG_NETWORKING=y
+CONFIG_MQTT_LIB=y
+CONFIG_MQTT_LIB_TLS=y
+CONFIG_NRF_CLOUD_MQTT=y
+CONFIG_NRF_MODEM_LIB=y
+
+# FOTA
+CONFIG_NRF_CLOUD_FOTA_FULL_MODEM_UPDATE=y
+CONFIG_ZCBOR=y
```

Comment:
you shouldnt need to set this or `CONFIG_MBEDTLS_SHA256_C`

Reply:
So I should remove that SHA256 config?

Reply:
when I set `CONFIG_NRF_CLOUD_FOTA_FULL_MODEM_UPATE=y` and build, CMake throws an error that the other 3 needs to be setup so that `FMFU_FDEV` can be `y`

Reply:
you need to set
`CONFIG_NRF_MODEM_LIB_SYS_INIT=n`

you should _not_ need to set:
`CONFIG_ZCBOR=y`
and
`CONFIG_MBEDTLS_SHA256_C=y`

Reply:
done, thanks